pkg/products/monitoring/prometheusRules.go

```diff
@@ -113,7 +112,7 @@ func (r *Reconciler) newAlertsReconciler(logger l.Logger, installType string) re
 							"sop_url": resources.SopUrlAlertsAndTroubleshooting,
 							"message": "Pod {{  $labels.namespace }} / {{  $labels.pod  }} has been trying to start for longer than 15 minutes - this could indicate a configuration error.",
 						},
-						Expr:   intstr.FromString("(kube_pod_container_status_waiting_reason{reason='ContainerCreating'} * on (namespace, namespace) group_left(label_monitoring_key) kube_namespace_labels{label_monitoring_key='middleware'}) > 0"),
+						Expr:   intstr.FromString("kube_pod_container_status_waiting_reason{reason='ContainerCreating',namespace=~'^redhat-rhoam-[a-zA-Z]+$'} > 0"),
```

Comment:
Three alerts above have hard-coded "redhat-rhoam-" prefix.
@laurafitzgerald Is that intentional?

Reply:
oh, I see your comments above... nvm